### ISSUE TITLE
Include precopiled bootstrap styles

### ISSUE THREAD

**Issue by jacekmichalski** [cscart/cscart-boilerplate#26]
Please include precompiled bootstrap.less and compile it on the fly. It will then allow usage and override of built-in bootstrap variables in styles.less such as @font-family-sans-serif etc and will provide cleaner style sheets.

**Comment by amelekesov** [comment]
Including precompiled bootstrap.less would lead to higher loads when clearing the cache.
I think it's better to add bootstrap-theme.css (https://github.com/twbs/bootstrap/blob/master/dist/css/bootstrap-theme.css) and write the necessary changes there.

**Comment by drahosistvan** [comment]
Higher load after clearing cache can be acceptable, because store owners clearing the cache once a week, or even rarely.
I support this request!

**Comment by jacekmichalski** [comment]
I agree. It takes me approximate 3s to compile css from less in my build system. I think that this is acceptable and would make the development easier. Adjusting bootstrap variables by including bootstrap-theme or by generating all over again bootstrap.css is much more time consuming.

**Comment by Collins-l1fe** [comment]
does this boilerplate theme work with paid add-ons from simtech and cs-cart?
